lib/Driver/ParseableOutput.cpp

```diff
@@ -73,6 +75,16 @@ namespace json {
       return seq[index];
     }
   };
+
+  template<>
+  struct ScalarTraits<llvm::sys::TimePoint<>> {
+    static void output(const llvm::sys::TimePoint<> &value, llvm::raw_ostream &os) {
+      using namespace std::chrono;
+      os << duration_cast<microseconds>(value.time_since_epoch()).count();
```

Comment:
`time_since_epoch` isn't necessarily the same on all systems. We should probably pick our own epoch.